web/src/components/organisms/SelectInput.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Select } from 'flowbite-react';
+
+const SelectInput = ({ errors, name, register, label, options, ...rest }) => {
+    const formRegister = (name && register) && { ...register(name) };
+    const error = errors?.[name]?.message || null;
+
+    return (
+        <div className="max-w-md grid grid-cols-6 sm:grid-cols-3 gap-4">
+            <div className="col-span-6 sm:col-span-3 h-12 md:h-full">
+                <div className="w-full h-full">
```

Comment:
Not all will use this uncessary classes, remove it and apply to where this component called
```suggestion
                <div className="w-full">
```